README.md

```diff
@@ -1,93 +1,55 @@
 [//]: # "This README.md file is auto-generated, all changes to this file will be lost."
-[//]: # "To regenerate it, use `npm run generate-scaffolding`."
+[//]: # "To regenerate it, use `python -m synthtool`."
```

Comment:
I think it's supposed to just be `synthtool`, and it yells at me when I try to run it like that 😆

Reply:
It *should* work either way. :thinking: `python -m synthtool` is what the synthtool README says.

Reply:
ahhh, I could be thinking of another tool - apologies for the confusion!